tests/integration/generate_scaled_javascript_project.py

```
@@ -0,0 +1,289 @@
+    shutil.copy2(SOURCE_PROJECT / "jsconfig.json", TARGET_PROJECT / "jsconfig.json")
+    shutil.copy2(SOURCE_PROJECT / "package.json", TARGET_PROJECT / "package.json")
+
+    # git init + commit
+    env = {
+        **os.environ,
+        "GIT_AUTHOR_NAME": "test",
+        "GIT_AUTHOR_EMAIL": "test@test.com",
+        "GIT_COMMITTER_NAME": "test",
+        "GIT_COMMITTER_EMAIL": "test@test.com",
+    }
+    subprocess.run(["git", "init"], cwd=TARGET_PROJECT, capture_output=True, check=True)
```

Ruff (0.15.1)

[error] Starting a process with a partial executable path

(S607)